project/src/com/google/daggerqueryui/index.html

```diff
@@ -0,0 +1,39 @@
+<!--Copyright 2020 Google LLC-->
+
+<!--Licensed under the Apache License, Version 2.0 (the "License");-->
+<!--you may not use this file except in compliance with the License.-->
+<!--You may obtain a copy of the License at-->
+
+<!--https://www.apache.org/licenses/LICENSE-2.0-->
+
+<!--Unless required by applicable law or agreed to in writing, software-->
+<!--distributed under the License is distributed on an "AS IS" BASIS,-->
+<!--WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.-->
+<!--See the License for the specific language governing permissions and-->
+<!--limitations under the License.-->
+
+<html>
+<head>
+    <title>Dagger Query UI</title>
+
+    <link rel="stylesheet" href="page-style.css">
+    <link rel="stylesheet" href="third-party/css/bootstrap.min.css">
+    <link rel="stylesheet" href="third-party/css/mdb.min.css">
+    <link rel="stylesheet" href="third-party/css/roboto-fonts.css">
+
+    <!-- JQuery -->
+    <script src="third-party/js/jquery.min.js"></script>
+    <!-- MDB core JavaScript -->
+    <script type="text/javascript" src="third-party/js/mdb.min.js"></script>
```

Comment:
nit: How about adding the `defer` attribute to prevent javascript download from blocking the page load? See https://stackoverflow.com/a/24070373

Reply:
Thanks for sharing, that's cool feature, I didn't know about `defer` before. I have added it ✅